packages/destination-actions/src/destinations/canny-functions/identify/index.ts

```diff
@@ -0,0 +1,57 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import { BASE_API_URL } from '../constants/api'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+import cloneWithDefinedProps from '../utils/cloneWithDefinedProps'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Identify',
+  description: 'Identify a user on Canny',
```

Comment:
Maybe we should go with a more descriptive description here. 
For example: 

```suggestion
  description: 'Create or update a user profile in Canny',
```